scripts/npm/link/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
```

Comment:
Mmm, good point. I really want to spend some time adding some unit testing soon. That would catch an issue like this.